cluster/manifests/01-vertical-pod-autoscaler/updater-deployment.yaml

```diff
@@ -25,7 +25,7 @@ spec:
       serviceAccountName: vpa-updater
       containers:
       - name: updater
-        image: container-registry.zalando.net/teapot/vpa-updater:v0.6.1-internal.16
+        image: container-registry.zalando.net/teapot/vpa-updater:{{.Cluster.ConfigItems.vertical_pod_autoscaler_version}}
```

Comment:
Note that there are 3 components in total, those also need to be update, but see my suggestion above :)

Reply:
Ah I see. the `recommender` and `admission-controller` as well. I'll update those with the new value types as well :)